scene-lib/src/scenelib/annotations/el/LocalLocation.java

```diff
@@ -1,91 +1,219 @@
 package scenelib.annotations.el;
 
+import org.objectweb.asm.Label;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
 import java.util.Objects;
 
+import org.checkerframework.checker.nullness.qual.Nullable;
+
 /**
  * A {@link LocalLocation} holds location information for a local
- * variable: slot index, scope start, and scope length.
+ * variable: slot index, scope start, and scope end.
  */
 public final class LocalLocation {
     /**
-     * The slot index of the local variable.
+     * The starts of the scopes of the element being visited.
+     * Used only for TypeReference#LOCAL_VARIABLE and TypeReference#RESOURCE_VARIABLE.
+     */
+    public final Label[] start;
+
+    /**
+     * The ends of the scopes of the element being visited.
+     * Used only for TypeReference#LOCAL_VARIABLE and TypeReference#RESOURCE_VARIABLE.
      */
-    public final int index;
+    public final Label[] end;
```

Comment:
Please define the fields in the same order as they are described in the documentation and passed as arguments to the constructor.  I think `index` should always be before the other two, but in any event please make them consistent.

Reply:
Sorry - same order as what documentation?

Reply:
The [class documentation](https://github.com/typetools/annotation-tools/pull/340/files#diff-00843e3e4ce33df32b9e700c8a6e584e19c4a850a1f5e70dd1e033aa2d7074bdR13), which says, "slot index, scope start, and scope end."